### PR TITLE
Bump the schema version that has missing column in audit table

### PR DESCRIPTION
**Pull request by chdietz**
Address and remove todo's in IT around created column
Now that all columns exist in audit table add test that no nulls for at least one actual to audit compare,
has no null in actual to ensure no columns are left off the insert in the trigger.

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -25,7 +25,7 @@ apply plugin: 'io.spring.dependency-management'
 // Set version dependencies. This allows the cli to force version, e.g. `-Pcommon=1.1.0-SNAPSHOT`
 
 String rdwCommonVersion = project.hasProperty("common") ? project.property("common") : "1.1.0-53"
-String rdwSchemaVersion = project.hasProperty("schema") ? project.property("schema") : "1.1.0-245"
+String rdwSchemaVersion = project.hasProperty("schema") ? project.property("schema") : "1.1.0-246"
```
Comment (chdietz) [inline build.gradle]: new schema adds `created` column accidentally left off of audit tables

File: exam-processor/src/test/java/org/opentestsystem/rdw/ingest/processor/repository/impl/ExamRepositoryIT.java [hunk 2]
```diff
@@ -731,6 +759,9 @@ public void itShouldAuditExamUpdates() {
                 "SELECT * FROM audit_exam WHERE exam_id = :exam_id",
                 new MapSqlParameterSource().addValue("exam_id", originalExamId));
 
+        // At least one comparison between audit and actual must have no null values in actual to assure all columns are audited.
+        secondExamValues.forEach((k, v) -> assertThat(v).isNotNull());
```
Comment (chdietz) [inline exam-processor/src/test/java/org/opentestsystem/rdw/ingest/processor/repository/impl/ExamRepositoryIT.java]: We want one test with all possible fields not null this way the compare to audit helps insure we did not leave off a column on the audit trigger

File: exam-processor/src/test/java/org/opentestsystem/rdw/ingest/processor/repository/impl/ExamRepositoryIT.java [hunk 3]
```diff
@@ -1143,7 +1183,6 @@ private void compareAuditExamClaimScoreToActual(final List<Map<String, Object>>
                             .addValue("exam_id", originalExamId)
                             .addValue("subject_claim_score_id", examClaimScoreValues.get("subject_claim_score_id")));
 
-            examClaimScoreValues.remove("created"); // todo remove after adding column to audit table
```
Comment (chdietz) [inline exam-processor/src/test/java/org/opentestsystem/rdw/ingest/processor/repository/impl/ExamRepositoryIT.java]: Three of these todo's are removed as the column now exists

File: exam-processor/src/test/java/org/opentestsystem/rdw/ingest/processor/repository/impl/ExamRepositoryIT.java [hunk 4]
```diff
@@ -834,7 +865,6 @@ public void itShouldAuditExamDeletes() {
 
     @Test
     public void itShouldAuditExamAvailableAccommodations() {
-        // todo: Need at one test per audit table where audited record has no null values to ensure no missing columns in trigger sql and audit table.
```
Comment (chdietz) [inline exam-processor/src/test/java/org/opentestsystem/rdw/ingest/processor/repository/impl/ExamRepositoryIT.java]: This todo is removed because we have this test on each audit table.

File: exam-processor/src/test/java/org/opentestsystem/rdw/ingest/processor/repository/impl/ExamRepositoryIT.java (hunk 3 above)
Comment (malaffoon) [reply exam-processor/src/test/java/org/opentestsystem/rdw/ingest/processor/repository/impl/ExamRepositoryIT.java]: 👍
